### PR TITLE
Deleted Actions

### PR DESCRIPTION
**Pull request by jokano12**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [ ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [ ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by jokano12** [comment]
not certain why there are 2 other commits from Jan. for different repositories, just ignore them.

File: certified-connectors/Africa's Talking Payments/apiDefinition.swagger.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{
+{{
```
Comment (lordlydon) [inline certified-connectors/Africa's Talking Payments/apiDefinition.swagger.json]: Hi @jokano12, can you check why is there a change in the definition for a different connector?

File: certified-connectors/Africa's Talking Payments/apiDefinition.swagger.json [hunk 2]
```diff
@@ -1208,4 +1208,4 @@
     }
   ],
   "tags": []
-}
\ No newline at end of file
+}
```
Comment (lordlydon) [inline certified-connectors/Africa's Talking Payments/apiDefinition.swagger.json:1211]: I still see a change, seems like a newline?

Comment (jokano12) [reply certified-connectors/Africa's Talking Payments/apiDefinition.swagger.json:1211]: I checked their Definition, and it shows it only has 1211 lines. shouldnt I just keep it? I added a line so its 1212, but i think i need to revert back.

Comment (lordlydon) [reply certified-connectors/Africa's Talking Payments/apiDefinition.swagger.json:1211]: Yeah, we need to make sure it doesn't show as a change when we make the PR to Microsoft's repo.
If nothing else works, you could download the file from https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa's%20Talking%20Payments/apiDefinition.swagger.json and replace the one you have.

**Comment by lordlydon** [comment]
I was checking and found a couple of definitions that aren't being used, so we can remove those too:

- ProspectData
- OpportunityData
- SubscriptionTypeData
- SponsorshipData
- PublicationTypeData
- CourseData
- MembershipTypeData
- MerchandiseData
- SessionData
- ContactImageData
- SpeakerData
- ProspectListData
- SpeakerTypeData
- SpeakerSaveData
- PresentationTopicData
- TrackData
- RoomData
- MembershipStatsStateData
- CityData
- PasswordData
- SaveOrganizationData
- RelationshipSaveData
- CommitteeMemberDetailData

File: certified-connectors/Africa's Talking Payments/apiDefinition.swagger.json (hunk 2 above)
Comment (jokano12) [reply certified-connectors/Africa's Talking Payments/apiDefinition.swagger.json:1211]: I still cant get it to remove the change.

**Comment by jokano12** [comment]
going to try to reopen pull request

**Pull request closed without merging**